tools/pelela-cli/templates/basic-converter/README.md

```
@@ -0,0 +1,52 @@
+# 🪴 Ejemplo de PelelaJS
+
+¡Bienvenido a tu nuevo proyecto generado con Pelela CLI! Este esqueleto básico está listo para que empieces a programar, e incluye configuraciones modernas (ESLint estricto, TypeScript) y routing por defecto.
+
+## 🚀 Cómo ejecutarlo
+
+1. **Instalar las dependencias:**
+   Este proyecto utiliza `pnpm` como gestor de paquetes. Para instalar todo lo necesario, ejecutá:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)